src/version.rs

```diff
@@ -300,16 +300,17 @@ pub fn dependencies(req: &mut Request) -> CargoResult<Response> {
 
 /// Handles the `GET /crates/:crate_id/:version/downloads` route.
 pub fn downloads(req: &mut Request) -> CargoResult<Response> {
+    let (offset, limit) = req.pagination(90, 100)?;
```

Comment:
I did it to be consistent with the other uses.  The advantage of `per_page` is the consistency with other endpoints. All in all, I am not strongly wedded to this solution.